inc/formanswer.class.php

```diff
@@ -44,6 +44,7 @@ class PluginFormcreatorFormAnswer extends CommonDBTM
 
    /**
     * Generated targets after creation of a form answer
+    * Populated only after creation of a FormAnswer in DB with add() method
```

Comment:
this property is used only to completesmart HTTP redirection after ceating a formanswer from a form. See front/form.form.php